phost/src/main.rs

```diff
@@ -221,15 +262,34 @@ async fn bridge(args: Args) -> Result<(), Error> {
     let mut info = req_decode("get_info", GetInfoReq {}).await?;
     if !info.initialized && !args.no_init {
         println!("pRuntime not initialized. Requesting init");
-        req_decode("init_runtime", InitRuntimeReq {
-            skip_ra: !args.ra,
-            bridge_genesis_info_b64: args.get_genesis()
-        }).await?;
+		let block = get_block_at(&client, Some(0)).await?.block;
+		let hash = client.block_hash(Some(subxt::BlockNumber::from(NumberOrHex::Number(0)))).await?;
+		let storage_key = StorageKey(GRANDPA_AUTHORITIES_KEY.to_vec());
+		let value = get_storage(&client, hash, storage_key.clone()).await.unwrap();
+
+		let proof = read_proof(&client, hash, storage_key).await.proof;
+		let mut prf = Vec::new();
+		for p in proof {
+			prf.push(p.to_vec());
+		}
+
+		let v: AuthorityList = VersionedAuthorityList::decode(&mut value.as_slice()).unwrap().into();
```

Comment:
```suggestion
		let v: AuthorityList = VersionedAuthorityList::decode(&mut value.as_slice())
			.expected("Failed to decode VersionedAuthorityList").into();
```

Reduce the usage of unwrap